cpp/test/spatial/spatial_data.h

```diff
@@ -0,0 +1,1674 @@
+#include <vector>
+
+namespace raft {
+namespace spatial {
+std::vector<float> spatial_data = {

```

Comment:
Could you add a comment just saying where these data come from, whether they're randomly generated or brought from some other source?